### DISCUSSION TITLE
Embed a 3D-Viewer dynamically

### DISCUSSION THREAD

**Discussion by nerudasson** [Budibase/budibase#12155] [Help] [answered]
Hello, 

I´m quite new to coding so please excuse my question. 

I want to embed a 3D-Viewer into Budibase to show STLs and update the view according to the tablerow selected.

How could I implement this? Can I use the Busibase Embed function to use a viewer like this https://github.com/ajhamdi/html_3dviewer ?

Any help is appreciated. Thanks!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @nerudasson,

This is a third party plugin, you'll need to import the plugin into your Budibase instance. You can read up on how to do this [here](https://docs.budibase.com/docs/custom-plugin#importing-plugins).
